pkg/client/egressfirewall.go

```diff
@@ -0,0 +1,50 @@
+package client
+
+import (
+	sdnapi "github.com/openshift/origin/pkg/sdn/api"
+)
+
+// EgressFirewallsInterface has methods to work with EgressFirewall resources
+type EgressFirewallsInterface interface {
+	EgressFirewall() EgressFirewallInterface
+}
+
+// EgressFirewallInterface exposes methods on egressFirewall resources.
+type EgressFirewallInterface interface {
+	Get(name string) (*sdnapi.EgressFirewall, error)
+	Create(sub *sdnapi.EgressFirewall) (*sdnapi.EgressFirewall, error)
+	Update(sub *sdnapi.EgressFirewall) (*sdnapi.EgressFirewall, error)
+}
+
+// egressFirewall implements EgressFirewallInterface interface
+type egressFirewall struct {
+	r *Client
+}
+
+// newEgressFirewall returns a egressFirewall
+func newEgressFirewall(c *Client) *egressFirewall {
+	return &egressFirewall{
+		r: c,
+	}
+}
+
+// Get returns information about a particular firewall
+func (c *egressFirewall) Get(name string) (result *sdnapi.EgressFirewall, err error) {
+	result = &sdnapi.EgressFirewall{}
+	err = c.r.Get().Resource("egressFirewalls").Name(name).Do().Into(result)
```

Comment:
I'm reading the kube api conventions doc [1] and at the end of the opening it mentions that collections should be lowercase and plural (e.g. egressfirewalls instead of egressFirewalls).  Do you know if origin follows the same convention? 

1: https://github.com/kubernetes/kubernetes/blob/master/docs/devel/api-conventions.md